tests/Unit/ApparentHorizons/Test_StrahlkorperGr.cpp

```diff
@@ -272,3 +341,37 @@ SPECTRE_TEST_CASE("Unit.ApparentHorizons.StrahlkorperGr.RicciScalar",
       },
       [](const size_t size) noexcept { return DataVector(size, 0.5); });
 }
+
```

Comment:
Sorry I missed this last time, the anonymous namespace should end here. That is, move the line `}  // namespace` to here, otherwise we risk linking errors in some cases. Just squash that :)